README.md

```diff
@@ -21,6 +21,8 @@ For local development, also install:
   * [yarn](https://yarnpkg.com/lang/en/docs/install/#mac-stable)
   * [Node.js](https://nodejs.org/en/)
   * [Java 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+  * [Docker sync](https://docker-sync.io)
```

Comment:
Should we update the Docker config suggestion on line 11?

Reply:
Done.